recipe/sched_yield.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/lib/easy_lock.h b/lib/easy_lock.h
+index 819f50ce8..e56384dac 100644
+--- a/lib/easy_lock.h
++++ b/lib/easy_lock.h
+@@ -40,6 +40,11 @@
+ #define curl_simple_lock atomic_bool
+ #define CURL_SIMPLE_LOCK_INIT false
+ 
++//Missing inclusion for s390x
```

Comment:
I would remove this comment, since the patch isn't (or doesn't have to be) s390x specific.

Reply:
When I removed the comment it broke the build. I ended up keeping the comment and generalizing it.